e2e-pw/src/oss/poms/modal.ts

```diff
@@ -29,15 +41,57 @@ export class ModalPom {
       return sampleId !== currentSampleId;
     }, currentSampleId);
 
-    return this.waitForSampleToLoad();
+    return this.waitForSampleToLoad(allowErrorInfo);
   }
 
-  async navigateNextSample() {
-    return this.navigateSample("forward");
+  async scrollCarousel(left: number = null) {
+    await this.groupCarousel.getByTestId("flashlight").evaluate((e, left) => {
+      e.scrollTo({ left: left ?? e.scrollWidth });
+    }, left);
   }
 
-  async navigatePreviousSample() {
-    return this.navigateSample("backward");
+  async navigateCarousel(index: number, allowErrorInfo = false) {
+    const looker = this.groupCarousel.getByTestId("looker").nth(index);
+    await looker.click({ position: { x: 10, y: 60 } });
+
+    return this.waitForSampleToLoad(allowErrorInfo);
+  }
+
+  async navigateSlice(
+    groupField: string,
+    slice: string,
+    allowErrorInfo = false
+  ) {
+    const currentSlice = await this.sidebarPom.getSidebarEntryText(
+      `sidebar-entry-${groupField}.name`
```

Comment:
nit: should this be `sidebar-entry-${groupField}-name` with a dash

Reply:
Ok, we would need to follow that formatting in the react component. But that would create collisions, though this is only for testing